src/EmotionRecognitionTask.ts

```
@@ -15,13 +15,13 @@ import { experimentSettingsJson } from './experimentSettings.ts';
 import { $Settings } from './schemas.ts';
 import { transformAndExportJson, downloadJson, transformAndDownload } from './dataMunger.ts';
 
-export default async function emotionRecognitionTask() {
+export default async function emotionRecognitionTask(onFinish?: (data: any) => void) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)